src/engine/BMBtnSkillRandomBMPentaskill.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * BMBtnSkillRandomBMPentaskill: Code specific to RandomBMPentaskill
+ *
+ * @author: james
+ */
+
+/**
+ * This class currently supports the special skills of RandomBMPentaskill, which has
+ * 4 normal dice and a swing die, with 5 skills appearing a total of 13 times on various dice.
```

Comment:
The actual balance of "how many instances of skills" may well need to be tweaked or balanced based on tester feedback anyway.